src/svelte/src/components/Header/fieldsets/FileMetrics.svelte

```diff
@@ -71,6 +72,9 @@ limitations under the License.
           if ('type' in msg.data.data) {
             $fileMetrics.type = msg.data.data.type
           }
+          if ('language' in msg.data.data) {
+            $fileMetrics.language = msg.data.data.language
```

Comment:
Honestly, I'm looking, but I'm not finding any code that does any language guessing calls here. 

Can you point out when and where exactly this code takes some action which provides the guess at the language?